### DISCUSSION TITLE
How can I extend gap between text and menu?

### DISCUSSION THREAD

**Discussion by watchstep** [nunocoracao/blowfish#656] [Q&A]
Hi, I'm using my blog website with the blowfish theme.

I want to extend gap between text and menu in the article.

current example:
![image](https://user-images.githubusercontent.com/88659167/230789723-456637b1-c876-4db8-aa01-4d305d1c0dcc.png)

wanted example:
![image](https://user-images.githubusercontent.com/88659167/230789792-d5f4209b-2523-4057-b145-912de0c41ff6.png)

If you know how to fix to make the website like the picture above, please help me..!

**Comment by nunocoracao** [comment]
No configuration for that. You can override the CSS or the actual partial. I would explore the single.html or the toc.html files.

**Comment by watchstep** [comment]
Thank you for your answer!!!
